schema/tables/screen_lock.yml

```diff
@@ -0,0 +1,18 @@
+name: screen_lock
```

Comment:
Could we change this name to `screenlock`? There is already such a table.

Reply:
It didn't previously exist in the fleet schema, but was part of the osquery schema. Just today we added the table to the fleet schema to make it more explicit about its limitations on Mac. https://github.com/fleetdm/fleet/pull/12432

Reply:
@lucasmrod and I had a discussion about this and keeping them separate since fleet wasn't able to return data for `screenlock`, I will bring this up again at standup

Reply:
Update: Speaking with @zhumo and @lucasmrod at standup, product decision to merge the ChromeOS screen_lock table with the macOS screenlock table even if Fleet doesn't support the macOS screenlock table. 

Reasoning: consistency, fewer caveats